src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
* Kindly Use class-based components here as stated in the project [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components.).

Reply:
Thank you for pointing it out. I updated `App` to class 
`Calculator` is already mentioned as a function.

https://github.com/eapenzacharias/magic-calculator/blob/e925ca4c6d58e396a79f30b95445900338bb07a0/src/components/Calculator.js#L7-L31